gallery/TOC.yml

```diff
@@ -9,37 +9,37 @@
     - name: Finding Items in the Gallery
       items:
         - name: Searching For Items
-          href: how-to/finding-items/search-syntax.md
+          href: how-to/finding-packages/search-syntax.md
         - name: Searching by PSEdition
-          href: how-to/finding-items/searching-by-psedition.md
+          href: how-to/finding-packages/searching-by-psedition.md
         - name: Filtering Search Results
-          href: how-to/finding-items/filtering-items.md
+          href: how-to/finding-packages/filtering-packages.md
     - name: Publishing to the Gallery
       items:
         - name: Creating and publishing items
-          href: how-to/publishing-items/publishing-an-item.md
+          href: how-to/publishing-packages/publishing-a-package.md
```

Comment:
The files were changed to 'an-package'.  I think this is more correct, but one or the other needs fixing.

Reply:
fixed